scripts/just.config.ts

```diff
@@ -19,15 +40,44 @@ export const build = () => {
     target: "es6",
   };
   return parallel(
-    esbuildTask({
-      ...baseEsbuildOptions,
-      format: "esm",
-      outExtension: { ".js": ".mjs" },
-    }),
-    esbuildTask({
-      ...baseEsbuildOptions,
-      format: "cjs",
-    })
+    series(
+      esbuildTask({
+        ...baseEsbuildOptions,
+        format: "esm",
+        outdir: "lib/mjs",
+        bundle: true,
+        plugins: [
+          {
+            name: "add-mjs",
+            setup(build) {
+              build.onResolve({ filter: /.*/ }, (args) => {
+                if (args.importer) {
```

Comment:
What's `args.importer`?

Reply:
This plugin runs whenever esbuild is trying to "bundle" the package. Importer checks that something is importing this package. We rewrite it to include '.js' extension then.

Reply:
Gotcha, ta